iac/terraform/aws/variables.tf

```diff
@@ -42,3 +47,8 @@ variable "combined_openapi_file" {
   description = "Combined OpenAPI file"
   default     = "openapi.json"
 }
+
+variable "from_email_address" {
+  type    = string
+  default = "no-reply"
```

Comment:
Pass this from terraform as full email address

Reply:
hardcoded email address.[Commit](https://github.com/CraftsmenLtd/BloodConnect/pull/113/commits/17da4268b8fa1b4b6806ef56f93e80823641c2ae)